src/AsyncGenerator.yml

```diff
@@ -0,0 +1,249 @@
+﻿solution:
+  filePath: ..\..\..\src\NHibernate.sln #wants to resolve from the execuing path.
```

Comment:
Thanks for reporting this, now is [fixed](https://github.com/maca88/AsyncGenerator/commit/e00704f9a6c8908c6183ccc0ae47c71edbb08a7f) and published.